src/scripts/main.js

```diff
@@ -1,11 +1,24 @@
 'use strict';
 
 function waitFor(element, eventName) {
-// write your code here
+  const resolver = (resolve, reject) => {
+    element.addEventListener(eventName, e => {
+      resolve(`
+        It was ${eventName} on the element: ${element.nodeName},
+        id: ${element.id}
```

Comment:
```suggestion
        id: ${element.id}.
```
I guess that's the reason of failing tests.
Make sure that it's equal to this text:
`It was ${eventName} on the element: ${element.nodeName}, id: ${element.id}.`